teaspoons-cli/teaspoons/cli.py

```diff
@@ -1,14 +1,29 @@
 # cli.py
 
-import typer
-from commands.auth import auth_app
-from commands.pipelines import pipelines_app
+import click
+import logging
 
-cli = typer.Typer()
-cli.add_typer(auth_app, name="auth", help="Authentication commands")
-cli.add_typer(pipelines_app, name="pipelines", help="Pipeline commands")
+from teaspoons import __version__
+from teaspoons.commands.auth_commands import auth
+from teaspoons.commands.pipelines_commands import pipelines
+
+
+# Context settings for commands, for overwriting some click defaults
+CONTEXT_SETTINGS = dict(help_option_names=["-h", "--help"])
```

Comment:
just curious, what does click default as the `help_option_names` ?  just `-h`?

Reply:
looks like only `--help` - without this:

```
 ✗ ./teaspoons -h
Usage: teaspoons [OPTIONS] COMMAND [ARGS]...
Try 'teaspoons --help' for help.

Error: No such option: -h
```